apps/scan/backend/src/app_usb_drive.test.ts

```diff
@@ -0,0 +1,23 @@
+import { withApp } from '../test/helpers/custom_helpers';
+
+test('getUsbDriveStatus', async () => {
+  await withApp({}, async ({ apiClient, mockUsbDrive }) => {
+    mockUsbDrive.removeUsbDrive();
+    await expect(apiClient.getUsbDriveStatus()).resolves.toEqual({
+      status: 'no_drive',
+    });
+    mockUsbDrive.insertUsbDrive({});
+    await expect(apiClient.getUsbDriveStatus()).resolves.toEqual({
+      status: 'mounted',
+      mountPoint: expect.any(String),
+      deviceName: 'mock-usb-drive',
+    });
+  });
+});
+
+test('ejectUsbDrive', async () => {
+  await withApp({}, async ({ apiClient, mockUsbDrive }) => {
+    mockUsbDrive.usbDrive.eject.expectCallWith().resolves();
+    await expect(apiClient.ejectUsbDrive()).resolves.toBeUndefined();
```

Comment:
do you need to check status after ejecting?

Reply:
I don't think so. Since the usb drive library is mocked, I don't think checking status after ejecting would actually test anything here.